exercitiu3/index.js

```diff
@@ -0,0 +1,6 @@
+const numbers = ['2', '10', '12', '20', '25']
+
+for (let i = 0; i < numbers.length; i += 1) {
```

Comment:
```suggestion
for (let i = 0; i < numbers.length; i++) {
```
(opt), e acelasi lucru doar ca se scrie mai usor.